src/tests/ftest/dfuse/read.py

```diff
@@ -23,7 +23,9 @@ def test_dfuse_pre_read(self):
         Read one large file entirely using pre-read.  Read a second smaller file to ensure that
         the first file leaves the flag enabled.
 
-        :avocado: tags=all,full_regression
+        TODO: Check this with 128k I/O size and non-128K I/O size.
+
+        :avocado: tags=all,daily_regression
```

Comment:
Just confirming that moving this from weekly to daily is intentional (I thought I had asked this before, but couldn't find the question in the ticket).  According to the [last weekly](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/weekly-testing/411/testReport/FTEST_dfuse/DFusePreReadTest/) it only takes a minute to run, so the move should be fine.